### PR TITLE
Initial version of capacities fetcher

### PR DESCRIPTION
**Pull request by msusicky**

### CODE REVIEW

File: app/fetcher/hospital_capacities_fetcher.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+from app.fetcher.fetcher import Fetcher
+from app.models import OckovaniSpotreba, OckovaciMisto, KapacityNemocnic
+
+
+class HospitalCapacitiesFetcher(Fetcher):
+    """
+    Class for updating capacities in hospitals.
+    """
+
+    USED_CSV = 'https://dip.mzcr.cz/api/v1/kapacity-intenzivni-pece-zdravotnicke-zarizeni-10-2021.csv'
```
Comment (jan-stanek) [inline app/fetcher/hospital_capacities_fetcher.py:13]: variable name :-)

File: app/fetcher/hospital_capacities_fetcher.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+    def __init__(self):
+        super().__init__(KapacityNemocnic.__tablename__, self.USED_CSV)
+
+    def fetch(self, import_id: int) -> None:
+        df = pd.read_csv(self._url)
+
+        df['luzka_standard_kyslik_kapacita_volna_covid_pozitivni'] = df[
```
Comment (jan-stanek) [inline app/fetcher/hospital_capacities_fetcher.py:21]: it should be possible to use something like this:
```
for col in df.columns:
  if col not in ['blacklisted_column']:
    df[col] = df[col].fillna('0').astype('int')
```

File: app/models.py [hunk 3]
```diff
@@ -1031,3 +1031,40 @@ class BezRegistraceFronta(db.Model):
 
     def __repr__(self):
         return f"<BezRegistraceFronta(id='{self.id}')>"
+
+class KapacityNemocnic(db.Model):
+    __tablename__ = 'kapacity_nemocnic'
+
+    datum = Column(DateTime, primary_key=True)
+    zz_kod = Column(Unicode, primary_key=True)
```
Comment (jan-stanek) [inline app/models.py:1039]: is it something different than nrpzs?

File: app/fetcher/hospital_capacities_fetcher.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+class HospitalCapacitiesFetcher(Fetcher):
+    """
+    Class for updating capacities in hospitals.
+    """
+
+    USED_CSV = 'https://dip.mzcr.cz/api/v1/kapacity-intenzivni-pece-zdravotnicke-zarizeni-10-2021.csv'
+
+    def __init__(self):
+        super().__init__(KapacityNemocnic.__tablename__, self.USED_CSV)
```
Comment (jan-stanek) [inline app/fetcher/hospital_capacities_fetcher.py:16]: it was updated after the midnight: `last-modified: Thu, 11 Nov 2021 00:00:02 GMT`

I'm not sure if we want to check modified date. If it's updated before the midnight, fetch will fail.

File: app/models.py (hunk 3 above)
Comment (msusicky) [reply app/models.py:1039]: Yes, it is ICZ

File: app/fetcher/hospital_capacities_fetcher.py (hunk 2 above)
Comment (msusicky) [reply app/fetcher/hospital_capacities_fetcher.py:21]: Ok, changed

**Review by jan-stanek — APPROVED** [review]

**Pull request merged**